.github/workflows/build.yml

```diff
@@ -22,3 +22,4 @@ jobs:
             target
           key: ${{ runner.os }}-cargo-${{ steps.rust_toolchain.outputs.rustc_hash }}
       - run: cargo test --all
+      - run: cargo run --bin rune -- scripts/*
```

Comment:
This won't run every script. It will currently only pick up the last one:

https://github.com/rune-rs/rune/blob/master/crates/rune-cli/src/main.rs#L141

Reply:
Ah, that's too bad. I'll make it a proper loop.

Reply:
I also realized that we don't set an exit status in case the script errors :)

Reply:
Just saw that too! Anyway, updated now with a fancy bash for-loop; hope that works on CI. Let's do further fixes for that in a future PR?